src/web/FriendlyFire.ts

```diff
@@ -35,7 +35,7 @@ if (!isElectron()) {
         navigator.registerProtocolHandler("web+friendlyfiregame", "https://play.friendlyfiregame.com/?s=%s");
     }
 
-    if ("serviceWorker" in navigator) {
+    if (process.env["MODE"] === "production" && "serviceWorker" in navigator) {
```

Comment:
Does this really work?
This code is executed within the browser's context after all...

... possible that it works and I simply don't get it, though. ;-)

Reply:
Webpack handles this with the DefinePlugin. process.env.MODE is defined in the webpack config so webpack knows how to resolve this into a simple object. It does not really use the process module of Node.js